andela-oakinseye/checkpoint1-inverted-index
# fixes for second feedback issues

#### What does this PR do?
This pull request fixes the issues raise during the second feedback after my checkpoint defences.

#### Description of Task to be completed?
A number of issues and recommendations have been resolved such as the suggestions to looking into an alternative method to using my test files rather than piping them.

I also changed the helper class name.

#### How should this be manually tested?
Use `gulp test` to test and look for locations to see where test files are being served from.

#### Any background context you want to provide?
N/A

#### Screenshots (if appropriate)
![screen shot 2016-12-07 at 10 35 28 pm](https://cloud.githubusercontent.com/assets/23452643/20987582/88011abc-bccd-11e6-9bef-9bb74e114898.png)

#### Questions:
N/A
